src/TLuaInterpreter.cpp

```diff
@@ -2971,6 +2971,26 @@ int TLuaInterpreter::setModulePriority(lua_State* L)
     return 0;
 }
 
+int TLuaInterpreter::getModuleSync(lua_State* L)
+{
+    if (!lua_isstring(L, 1)) {
+        lua_pushstring(L, "getModuleSync: Module must be a string");
+        return lua_error(L);
+    }
+    
+    QString moduleName = lua_tostring(L, 1);
```

Comment:
Only this is wrong now - you have to convert from the UTF-8 that is used within the Lua sub-system - the line needs to be:
```cpp
    QString moduleName = QString::fromUtf8(lua_tostring(L, 1));
```

Reply:
Is there any reason why this is true here, but not so in getModulePriority? 
`QString moduleName;
    if (!lua_isstring(L, 1)) {
        lua_pushstring(L, "getModulePriority: Module be be a string");
        lua_error(L);
        return 1;
    } else {
        moduleName = lua_tostring(L, 1);
    }`
or are we just adhering to better standards?

Reply:
Like in Discord you can use three \` followed by a code type (e.g. `lua` or `c`) before *several* lines of text (and another three \` afterwards) to format a block of code for display here on GitHub.  As such I am not sure what you are asking... :confused: